api.md

```diff
@@ -431,12 +434,104 @@ PostGIS geospatial indexes).  During testing, you might check your
+| `chunk_time_interval` | Interval in event time that each chunk covers. Must be > 0. As of TimescaleDB v0.11.0, default is 7 days, unless adaptive chunking (DEPRECATED)  is enabled, in which case the interval starts at 1 day. For previous versions, default is 1 month. |
+| `create_default_indexes` | Boolean whether to create default indexes on time/partitioning columns. Default is TRUE. |
+| `if_not_exists` | Boolean whether to print warning if table already converted to hypertable or raise exception. Default is FALSE. |
+| `partitioning_func` | The function to use for calculating a value's partition.|
+| `associated_schema_name` | Name of the schema for internal hypertable tables. Default is "_timescaledb_internal". |
+| `associated_table_prefix` | Prefix for internal hypertable chunk names. Default is "_hyper". |
+| `migrate_data` | Set to `true` to migrate any existing `main_table` data to chunks in the new hypertable. A non-empty table will generate an error without this option. Note that, for large tables, the migration might take a long time. Defaults to false. |
+| `time_partitioning_func` | Function to convert incompatible primary time column values to compatible ones. The function must be `IMMUTABLE`. |
+| `replication_factor` | This will determine the number of data nodes each incoming write is written to.  The default value is 1 and this is required to be 1 or greater, but note that values greater than 1 are not recommended in the current implementation. |
+| `data_nodes` | This is the set of data nodes that will be used for this table.  If this is not present then all data nodes known by this instance will be used to distribute the hypertable. | 
+| `chunk_target_size` | DEPRECATED - The target size of a chunk (including indexes) in `kB`, `MB`, `GB`, or `TB`. Setting this to `estimate` or a non-zero chunk size, e.g., `2GB` will enable adaptive chunking (a DEPRECATED feature). The `estimate` setting will estimate a target chunk size based on system information. Adaptive chunking is disabled by default. |
+| `chunk_sizing_func` | DEPRECATED - Allows setting a custom chunk sizing function for adaptive chunking (a DEPRECATED feature). The built-in chunk sizing function will be used by default. Note that `chunk_target_size` needs to be set to use this function.  |
+
+#### Returns
+
+|Column|Description|
+|---|---|
+| `hypertable_id` | ID of the hypertable in TimescaleDB. |
+| `schema_name` | Schema name of the table converted to hypertable. |
+| `table_name` | Table name of the table converted to hypertable. |
+| `created` | True if the hypertable was created, false when `if_not_exists` is true and no hypertable was created. |
+
+#### Sample Usage [](create_distributed_hypertable-examples)
+
+Create a table `conditions` which will be partitioned across data
+nodes by the 'location' column.  Note that the number of space
+partitions is automatically equal to the number of data nodes in this
```

Comment:
number of configured data nodes in the database

Reply:
Went a little more verbose with "number of data nodes assigned to this hypertable (all configured data nodes in this case, as `data_nodes` is not specified)."